UnattendResources/Logon.ps1

```diff
@@ -228,6 +260,14 @@ try
     $Host.UI.RawUI.WindowTitle = "Running SetSetupComplete..."
     & "$programFilesDir\Cloudbase Solutions\Cloudbase-Init\bin\SetSetupComplete.cmd"
 
+    #Apply local security settings
+     
+    $Host.UI.RawUI.WindowTitle = "Applying Local Security setting..."

```

Comment:
This is a special policy used only in a specific environment, so we cannot add this part to the mainstream code.
Is this security policy required in your scenario?